examples/deno.jsonc

```diff
@@ -1,6 +1,31 @@
 {
   "imports": {
```

Comment:
How about the root import `@typegrqph/index.ts`?

Reply:
@luckasRanarison Line 5?